packages/terra-avatar/docs/avatar.md

```diff
@@ -1,6 +1,6 @@
 # Terra Avatar
 
-The `Avatar` variant represents a person - it displays an image or initials in a circular frame. If neither are provided, a fallback user icon displays. This is the default export of the `terra-avatar` package.
+The `Avatar` variant represents a person - it displays an image or initials in a circular frame. If valid image is not provided and initials is provided, then fallback initial displays. If neither are provided, a fallback user icon displays. This is the default export of the `terra-avatar` package.
```

Comment:
Based on this description, is this still a breaking change? Initials are not required and if not supplied the component still has the default behaviour?

Reply:
Yes but if initials are supplied fallback will happen to initials instead of user Icon which was not the behaviour  before these code changes. So I Think it is a breaking change.

Reply:
I don't see that as breaking at all. The api does not change and consumers would get the behavior we expect.

Reply:
I see you are setting initials to be required. This paragraph needs to be updated.

```suggestion
The `Avatar` variant represents a person - it displays an image or initials in a circular frame. If valid image is not provided then the avatar falls back to displaying initials. This is the default export of the `terra-avatar` package.
```

Reply:
> I don't see that as breaking at all. The api does not change and consumers would get the behavior we expect.

In scenario : user would be expecting avatar to display user-icon when image fails to load but with this code changes it will display initials when image fails to load instead of displaying user-Icon. (this was something which was not happening before) so will these not be considered as breaking change..?